airflow/providers/http/hooks/http.py

```diff
@@ -186,8 +186,8 @@ def run_and_check(
 
         # Send the request.
         send_kwargs = {
-            "timeout": extra_options.get("timeout"),
             "allow_redirects": extra_options.get("allow_redirects", True),
+            "timeout": extra_options.get("timeout"),
```

Comment:
Hmm. What was the problem here :) ?

Reply:
Does not seem it fixes anything

Reply:
I get the error
```
Argument 2 to "send" of "Session" has incompatible type "**Dict[str, Optional[Any]]"; expected "bool"
```
And these parameters switch helped avoid the error. Very strange

Reply:
Annotate `send_kwargs` as `Dict[str, Any]` instead.

Reply:
Thanks, @uranusjr. It also works. Made changes